php/README.md

```diff
@@ -1,19 +1,8 @@
-# Google Cloud Vision API PHP examples
+![status: inactive](https://img.shields.io/badge/status-inactive-red.svg)
 
-This directory contains [Cloud Vision API](https://cloud.google.com/vision/) PHP samples.
+The Google Cloud Vision API PHP samples have moved. This directory is no longer
+actively developed or maintained.
 
-
-## Prereqisites
-1. [Sign up for Google Cloud Platform Account](http://cloud.google.com)
-2. [Enable Cloud Vision API](https://cloud.google.com/vision/docs/getting-started)
-3. [Get App Engine SDK for PHP](https://cloud.google.com/appengine/downloads#Google_App_Engine_SDK_for_PHP)
-
-
-## Setup
-1. Alter creds.php with values that make sense to your project.
-2. Create a project using GoogleAppEngineLauncher from SDK
-3. Copy files to GoogleAppEngineLauncher project directory
-4. Run
-
-
-"This is not an official Google product."
\ No newline at end of file
+For new work on this check out the
+[vision samples](https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/vision)
```

Comment:
We should point to `/vision/api`, instead of to the intermediate vision directory